system/include/emscripten/console.h

```diff
@@ -20,6 +20,12 @@ void emscripten_console_logf(const char *utf8String, ...) __attribute__((__forma
 void emscripten_console_warnf(const char *utf8String, ...) __attribute__((__format__(printf, 1, 2)));
 void emscripten_console_errorf(const char *utf8String, ...)__attribute__((__format__(printf, 1, 2)));
 
+// Write to the out() and err() hooks directly.
+// out() and err() are defined in shell.js.
```

Comment:
Perhaps say something like.  "out() and err() are defined in shell.js and have different behaviour from console.log/error.  For example under node they write the stderr and stdout streams which, especially in workers, are more direct ways to write output".

Reply:
I kind of wish we had better names for these ... but I can't think of any.

Reply:
Done.